buildenv/jenkins/aqaTestPipeline.groovy

```diff
@@ -40,7 +40,11 @@ JDK_VERSIONS.each { JDK_VERSION ->
         if (SDK_RESOURCE == "customized" ) {
             if (params.TOP_LEVEL_SDK_URL) {
                 // example: <jenkins_url>/job/build-scripts/job/openjdk11-pipeline/123/artifact/target/linux/aarch64/openj9/*_aarch64_linux_*.tar.gz/*zip*/openj9.zip 
-                download_url = params.TOP_LEVEL_SDK_URL + "/artifact/target/${os}/${arch}/${params.VARIANT}/*.tar.gz/*zip*/${params.VARIANT}.zip"
+                if (os.contains("windows")) {
+                    download_url = params.TOP_LEVEL_SDK_URL + "/artifact/target/${os}/${arch}/${params.VARIANT}/*.zip/*zip*/${params.VARIANT}.zip"
+                } else {
+                    download_url = params.TOP_LEVEL_SDK_URL + "/artifact/target/${os}/${arch}/${params.VARIANT}/*.tar.gz/*zip*/${params.VARIANT}.zip"
+                }
```

Comment:
Can we use a variable for `*.zip` or `*.tar.gz` value, so that we do not need to set download_url twice? 

```
def filter = "*.tar.gz"
if (os.contains("windows")) {
   filter = "*.zip"
}
download_url = params.TOP_LEVEL_SDK_URL + "/artifact/target/${os}/${arch}/${params.VARIANT}/${filter}/*zip*/${params.VARIANT}.zip"
```